adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -209,12 +214,12 @@ def __init__(
         self.on_unsubscribe = None
 
     # pylint: disable=too-many-branches
-    def _get_connect_socket(self, host, port, *, timeout=1):
+    def _get_connect_socket(self, host, port, *, timeout):
```

Comment:
Not really a strong request, but I do like having having keyword arguments have defaults where possible, so that not having to call them out works.  I would added the default value back for that reason.

Reply:
Makes sense, added back.